hack/generate-test-certs.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Note to self; probably better to replace this with https://github.com/FiloSottile/mkcert